.eslintrc

```diff
@@ -23,29 +29,27 @@
     'prefer-rest-params': 2,
     'quote-props': [2, 'as-needed', { 'keywords': true }],
     'space-unary-ops': 2,
-    'max-len': [0],  // No agreed upon max length and react lines get > 100 chars easily.
+    'max-len': [2, 250],  // TODO: make smaller
     'arrow-parens': 0, // We don't have an agreed upon pattern here
-    'no-underscore-dangle': 0, // We have build flags that use this
+    'no-plusplus': 0,
+    'no-underscore-dangle': 0,
+    'import/no-extraneous-dependencies': 0,
 
     // Disabled rules with rationale.
     'react/no-multi-comp': 0,      // Leave organization to code reviewer discretion.
     'react/prefer-stateless-function': 0, // Leave statelessness to code reviewer discretion.
-
     'react/jsx-closing-bracket-location': [2, 'after-props'],
     'react/jsx-equals-spacing': 2,
     'react/jsx-key': 2,
-    'react/jsx-no-bind': [0, {'ignoreRefs': true}],  // TODO: Enable after fixing.
     'react/jsx-pascal-case': 2,
     'react/sort-prop-types': [0, {'callbacksLast': true, 'requiredFirst': true}],  // TODO(awong): Too hard to turn on.
     'react/jsx-sort-props': [0, {'callbacksLast': true, 'shorthandFirst': true}],  // TODO(awong): Too hard to turn on.
     'react/jsx-no-duplicate-props': 2,
     'react/no-danger': 2,
     'react/no-deprecated': 2,
     'react/no-direct-mutation-state': 2,
-    'react/no-string-refs': 0,       // TODO(awong): Enable.
```

Comment:
There are a bunch of things that have been deleted here and I'm wondering why? Some of them are things I enabled in the starter app, including `no-string-refs` and `jsx-indent-props`.

Reply:
The `0` means disable, so deleting this line should turn it on? Or does it need to be explicitly set to `2`? I may have erroneously assumed they were disabling it since they did it explicitly here.

Reply:
Oh, good question. I guess I assumed it needs to be explicitly set to `2` because many of the other rules are configured that way, but I could be wrong! Reading the docs it's not totally clear, although [this page](https://eslint.org/docs/rules/) says that "No rules are enabled by default", so perhaps it does need to be explicitly set with `2`?

Reply:
The presets like `airbnb` turn on a lot of stuff so I assumed it was being explicitly turned off here to override that. There are comments to that effect. I can try turning it on explicitly to see if we start getting errors but would like to avoid leaving that in if the default is already okay.

Reply:
If you remove it, it'll fall back to the airbnb config value.

Reply:
Ah okay, I see that now. Sounds good.

Reply:
@annekainicUSDS I checked these properties and they *are* enabled by default in the Airbnb setup. Anything else need to be done here?